java-extensions/jni-connector/src/main/java/com/starrocks/jni/connector/OffHeapTable.java

```diff
@@ -149,66 +112,54 @@ public int getNumRows() {
 
     public long getMetaNativeAddress() {
         meta.appendLong(numRows);
-        for (int i = 0; i < types.length; i++) {
-            OffHeapColumnVector.OffHeapColumnType type = types[i];
-            OffHeapColumnVector column = vectors[i];
-            if (OffHeapColumnVector.isArray(type)) {
-                meta.appendLong(column.nullsNativeAddress());
-                meta.appendLong(column.arrayOffsetNativeAddress());
-                meta.appendLong(column.arrayDataNativeAddress());
-            } else {
-                meta.appendLong(column.nullsNativeAddress());
-                meta.appendLong(column.valuesNativeAddress());
-            }
+        for (OffHeapColumnVector v : vectors) {
+            v.updateMeta(meta);
         }
         return meta.valuesNativeAddress();
     }
 
-    /**
-     * For test only
-     */
-    public void show(int limit) {
+    public void print(int rowLimit) {
```

Comment:
add comments of test only

I think function name `show()` is more appropriate and SQL-style